assets/txs.js

```diff
@@ -435,5 +435,135 @@ export default [
         fee: {}
       }
     }
+  },
```

Comment:
We don't have a top-level directory named `assets`.  Did you intend for this file to go somewhere else?